### DISCUSSION TITLE
Different Prices on the page and checkout

### DISCUSSION THREAD

**Discussion by hakikihacioglu** [primefaces/community#2718] [PrimeNG Blocks] [answered]
hi, don't know if this is the right place, but here is my question: I wanted to buy the prime blocks but the prices differ. Can anyone help?

<img width="744" alt="Screenshot 2024-09-19 at 11 49 25" src="https://github.com/user-attachments/assets/67b79594-3320-41b9-a2c5-afc28082992d">
<img width="456" alt="Screenshot 2024-09-19 at 11 45 06" src="https://github.com/user-attachments/assets/e81abb32-a447-4fd0-9b54-e779ac1e712e">

**Comment by mehmetcetin01140** [comment, marked as answer]
Hi, the prices have not been updated due to an issue during deployment. The current price is $199. Thank you for your feedback. We will update the price immediately.

**Reply by hakikihacioglu** [reply]
Dear Mehmet,

I believe I still deserve the €99 price, as I brought the problem to your attention. As you know, the customer is always right.

Cheers from Paris,
Hasan
